dartagnan/src/main/java/com/dat3m/dartagnan/verification/solving/RefinementSolver.java

```diff
@@ -496,6 +495,10 @@ private static RefinementModel generateRefinementModel(Wmm original) {
                 if (name.equals(DATA) || name.equals(CTRL) || name.equals(ADDR) || name.equals(CRIT)) {
                     constraintsToCut.add(c);
                 }
+            } else if (c instanceof Definition def && def instanceof Fences) {
+                // (iii) continued: fencerel(F) is unsupported in CAAT
+                //  It should get rewritten to "po;[F];po" but if it was not, we cut it instead.
```

Comment:
"It should be rewritten" -> "It should be rewritten by the NormalizeFenceRelations pass"

Reply:
I will add that it should get rewritten by "some pass". If I add the concrete pass name, we will 100% forget to update it if the pass name ever changes :P.